host/src/FunctionsNetHost/FunctionsNetHost.csproj

```diff
@@ -28,7 +28,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />
-    <PackageReference Include="System.Text.Json" Version="8.0.0-preview.4.23259.5" />
+    <PackageReference Include="System.Text.Json" Version="8.0.4" />
```

Comment:
@kshyju Is it okay if we update this? Any particular reason for keeping a preview version here?